ts/sagas/wallet.ts

```diff
@@ -171,6 +173,13 @@ function* fetchWallets(
   }
 }
 
+function* addCreditCard(
+  creditCard: CreditCard,
+  pagoPaClient: PagoPaClient,
+  walletToken: string,
+  token: string
```

Comment:
since we have some many tokens hanging around, better give more specific names, also may be worth to rely on tagged types to avoid using one kind of token in place of another by mistake